packages/react-scripts/template/src/serviceWorker.js

```diff
@@ -63,8 +63,8 @@ function registerValidSW(swUrl, config) {
         if (installingWorker == null) {
           return;
         }
-        installingWorker.onstatechange = () => {
-          if (installingWorker.state === 'installed') {
+        installingWorker.onstatechange = stateChangeEvent => {
```

Comment:
What is the purpose of this change?

Reply:
See this comment: https://github.com/facebook/create-react-app/issues/5316#issuecomment-427707302